components/director/cmd/director/main.go

```diff
@@ -103,7 +103,7 @@ type config struct {
 	TenantMappingEndpoint         string `envconfig:"default=/tenant-mapping"`
 	RuntimeMappingEndpoint        string `envconfig:"default=/runtime-mapping"`
 	AuthenticationMappingEndpoint string `envconfig:"default=/authn-mapping"`
-	OperationEndpoint             string `envconfig:"default=/operations"`
+	OperationEndpoint             string `envconfig:"default=/last_operation"`
```

Comment:
The PUT API URL Path of the Operations API should not be changed - it should remain `/operation`.
Introduce `LastOperationEndpoint string` for the last operation GET API